components/dashboard/src/App.tsx

```diff
@@ -345,10 +362,73 @@ function App() {
         false,
     );
 
+    const acceptAllCookies = () => {
+        GitpodCookie.setCookie(ANALYTICAL_COOKIES, "true");
```

Comment:
@laushinka It would be nice if we would limit the amount of cookies involved.

I imagined we'd use a single cookie that we use to store a `JSON.stringify`d (propably even `base64` encoded) shape of this form to:
```
{
  analytics: bool | undefined;
  targeting: bool | undefined;
}
```

Reply:
Hmm, even when stringified in our code, it would end up being two different cookies in the browser if I understand correctly?

Reply:
> it would end up being two different cookies in the browser if I understand correctly?

The idea is to just have one cookie. basically doing:
```
const consent = { analytics: true, targeting: false };
ConsentCookie.set(domain, JSON.stringify(consent));
...
const consent = JSON.parse(ConsentCookie.get(domain));
```

Reply:
Have synced with @geropl and will adjust so that it's one cookie with an object for a value. Something like:
```
"gitpod-io-consent": {
    necessary: true,
    analytical: true,
    targeting: true
}
```

Reply:
cc @nisarhassan12

Reply:
The change to object was trickier than anticipated, because special characters of the values end up being encoded by the libraries. Ideally @nisarhassan12 and I would like the cookies to already be parsed and human-readable, but we decided to leave it to the consumer of the value, i.e. website and other parts of the app code, to parse it.

Reply:
✅